source/Playnite/ViewModels/MainViewModelBase.cs

```diff
@@ -567,7 +567,7 @@ public async Task UpdateLibrary(bool metaForNewGames, bool updateIntegrations, b
 
                 if (AppSettings.ScanLibInstallSizeOnLibUpdate)
                 {
-                    UpdateLibraryInstallSizes(token);
+                    UpdateGamesInstallSizes(token, Database.Games.ToList(), LOC.ProgressScanningGamesInstallSize);

```

Comment:
I don't think .ToList is necessary here. Change the argument type to IEnumerable.

Reply:
You are right, done in https://github.com/JosefNemec/Playnite/pull/3100/commits/f431896072f45a32c84b07adad01df74d53de42d